src/pages/solve-overview/index.js

```diff
@@ -76,16 +77,27 @@ export default Vue.component('solve-overview-content', {
     async filterByTopic() {
       const topic = window.rsSolveFilterTopic;
       if (typeof topic !== 'undefined') {
-        const cats = await this.getCategories();
-        const catData = _.find(cats, c => c.tid === topic);
-        if (catData) {
-          this.topic.header = catData.name;
-          this.topic.desc = catData.description;
+        // seperate syndicated and tid categories here
+        if (this.isSyndicatedPage) {
+          this.topic.header = 'Mike\'s Unprepaired Syndicated Title';
+          this.topic.desc = 'Lorem Ipsum';
```

Comment:
Oh actually, until we get the follow-up ticket in to use taxonomy here, we need to force this text through the translation filter.

Reply:
This has been updated